packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -570,7 +570,7 @@ the generated files). Instead, it is ${JSON.stringify(pj.main)}`);
 
     const logger = new Logger(this.loggerPort);
     this.loggers.push(logger);
-    await logger.start();
+    const currLoggerPort = await logger.start();
```

Comment:
```ts
const logger = new Logger();
this.loggers.push(logger);
this.loggerPort = await logger.start(this.loggerPort);
```

Modify `Logger` Class, move `loggerPort` param to `start` function.

Reply:
Made these changes, updated `Logger` class for `start` function with `loggerPort` as param.